docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: /en/sql-reference/functions/time-series-functions
+sidebar_position: 140
+sidebar_label: Time Series
+---
+
+# Time Series Functions
+
+Below functions are used for time series anlaysis.
+
+## seriesPeriodDetect
```

Comment:
You are more of an expert with this but are there other viable algorithms to detect periods in time series? E.g. algorithms with different trade-offs? E.g. (*) ... In that case, someone else may want to implement them later, and it would be more consistent to have the algorithms name appended to the function name, e.g. `seriesPeriodDetectFFT()`. (But I am really not sure here, kindly check)

(*)
https://www.cs.purdue.edu/homes/ake/pub/Tkde-0129-0504-1.pdf
https://stats.stackexchange.com/questions/16117/what-method-can-be-used-to-detect-seasonality-in-data
https://project.inria.fr/aaltd19/files/2019/08/AALTD_19_Boussard.pdf
https://www.sciencedirect.com/science/article/abs/pii/S0031320323000560

Reply:
You are right, there are other are other algorithms. I will modify the method name.